crates/common/src/config.rs

```diff
@@ -250,7 +296,8 @@ mod test {
     ) {
         let peer_config = ProtocolConfig::builder()
             .max_sent_data(max_sent_data)
-            .max_recv_data(max_recv_data)
+            .max_recv_data_online(max_recv_data)
+            .max_deferred_size(0)
```

Comment:
can we also include this `max_deferred_size` value as part of the rstest case arguments? so that it's clear that now there re 3 arguments that will affect the check result